### PR TITLE
Docs for Organizations Phase I

### PR DESCRIPTION
**Pull request by bgrenon**
Previews:
- [Accounts](https://neon-next-git-bgrenon-orgs-ea-neondatabase.vercel.app/docs/manage/accounts) page (explaining basics between personal and orgs)
- [Geting started](https://neon-next-git-bgrenon-orgs-ea-neondatabase.vercel.app/docs/manage/organizations): intro page to Orgs features
- [Manage organizations](https://neon-next-git-bgrenon-orgs-ea-neondatabase.vercel.app/docs/manage/orgs-manage): various action how-tos
- [Transfer projects](https://neon-next-git-bgrenon-orgs-ea-neondatabase.vercel.app/docs/manage/orgs-project-transfer): all transfer related methods on one page
- [API actions](https://neon-next-git-bgrenon-orgs-ea-neondatabase.vercel.app/docs/manage/orgs-api) 
- [CLI actions](https://neon-next-git-bgrenon-orgs-ea-neondatabase.vercel.app/docs/manage/orgs-cli)

### CODE REVIEW

File: content/docs/manage/accounts.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+Neon offers two account types for you to choose from:
+
+- **Personal account**
+- **Organization account** (Early Access)
+
+## Personal accounts
+
+When you first sign up with Neon, whether you're signing up on your own or you're invited to join, you start with a personal account. During your [onboarding](/docs/get-started-with-neon#step-2---onboarding-in-the-neon-console)), you are prompted to create your first personal project that you can get started with.
```
Comment (danieltprice) [inline content/docs/manage/accounts.md:14]:
```suggestion
When you first sign up with Neon, whether you're signing up on your own or you're invited to join, you start with a personal account. During your [onboarding](/docs/get-started-with-neon/signing-up#step-2-onboarding-in-the-neon-console), you are prompted to create your first personal project that you can get started with.
```

File: content/docs/manage/accounts.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+If you want to work with other people on this project, your options are:
+
+- [Share](/docs/guides/project-sharing-guide) the project with another Neon user
+- [Transfer](/docs/manage/orgs-project-transfer) the project from a personal to an Organization-level project, where you can then invite other users
+
+## Organization accounts
+
+<EarlyAccess/>
+
+Any Neon user in our Early Access Program can [create an Organization account](/docs/manage/organizations#create-an-organization), which allows for sharing projects across the various members that make up your team. Whether you create an organization or are invited to join one, you still retain your personal account, letting you manage personal projects independently of any organizations you belong to.
```
Comment (danieltprice) [inline content/docs/manage/accounts.md:29]:
```suggestion
Any Neon user in our Early Access Program can [create an Organization account](/docs/manage/organizations#create-an-organization), which allows you to share projects with your team members. Whether you create an organization or are invited to join one, you still retain your personal account, letting you manage personal projects independently of any organizations you belong to.
```

File: content/docs/manage/organizations.md [hunk 3]
```diff
@@ -1,451 +1,60 @@
...
-```
-
-For more detail, see [Neon CLI commands - orgs](/docs/reference/cli-orgs). We'll update this page as new management options become available.
-
-## Feature limitations
-
-As we continue to refine our Organizations feature during this private preview phase, please remember that these features are only available under paid account plans. Here are some temporary limitations you should be aware of:
+As we continue to refine our Organizations feature during this Early Access phase, please remember that these features are only available under paid account plans. Here are some temporary limitations you should be aware of:
 
 - **Integration limitations** — You cannot install new Vercel integrations on organization-owned projects. However, existing integrations will continue to work on projects transferred from personal to the organization account.
```
Comment (danieltprice) [inline content/docs/manage/organizations.md:51]: Should we add these limitations to the Vercel and GitHub integration pages as well? Might deflect some issues.

File: content/docs/manage/orgs-cli.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+├────────────────────────┼──────────────────┤
+│ org-ocean-art-12345678 │ Example Org      │
+└────────────────────────┴──────────────────┘
+```
+
+See [Orgs - CLI](/docs/reference/cli-orgs) to learn more.
+
+## Manage projects within an Organization
+
+The Neon CLI `projects` command now supports an `--org-id` option. This allows you to list or create projects within a specified organization.
```
Comment (danieltprice) [inline content/docs/manage/orgs-cli.md:38]:
```suggestion
The Neon CLI `projects` command supports an `--org-id` option. This allows you to list or create projects within a specified organization.
```

**Review by danieltprice — APPROVED** [review]
Looks great, Barry! That's a lot of content.
Just a few minor comments.

File: content/docs/manage/organizations.md (hunk 3 above)
Comment (bgrenon) [reply content/docs/manage/organizations.md]: Yes, we can do that in a follow-up issue where we mention orgs in various places outside of the orgs content.

**Pull request merged**